README.md

```diff
@@ -159,6 +159,23 @@ To get sample traces to try run the following command. The traces will be stored
 yarn download:sample-traces
 ```
 
+## Running UI tests
+
+To run the UI tests locally, use the following commands.
+
+```bash
+yarn
+yarn vsce:package
+yarn download:sample-traces
+yarn download:server
+yarn start:server & # or run in a separate shell
+yarn download:openvscode-server
+yarn configure:openvscode-server
+yarn start:openvscode-server & # or run in a separate shell
+yarn playwright install --with-deps
+yarn playwright test
+```
```

Comment:
What are the steps to execute when updating the source code or adding tests. I assume that not all steps need to be executed then. I'm not clear what to execute to minimise the time to re-run the test. Could you please add another step-by-step for this?

Reply:
I will split the setup tasks in a clearer way in the README.